pkg/proc/interface.go

```diff
@@ -134,3 +139,18 @@ func (cctx *ContinueOnceContext) GetManualStopRequested() bool {
 	defer cctx.StopMu.Unlock()
 	return cctx.manualStopRequested
 }
+
+// NumProcs returns the number of processes in the target group being resumed.
+func (cctx *ContinueOnceContext) NumProcs() int {
```

Comment:
This feels like it should be on `TargetGroup`.

Reply:
If we want to prevent backends from messing with the `TargetGroup` too much, we could still add an exported field in `ContinueOnceContext` but limit it via an interface since we're already storing the target group there, just not exported for the backends, and it really is only there to implement these methods.

```
type TargetGroupProcs interface {
  NumProcs()
  Proc(i int)
  AddTarget(t)
}

type ContinueOnceContext struct {
  ...
  TargetGroup TargetGroupProcs
}
```

Reply:
> If we want to prevent backends from messing with the TargetGroup too much

Rather than backends messing with the TargetGroup my concern is exposing extra things as the public interface of TargetGroup. Users of TargetGroup don't need to call `Proc(i)` and get a ProcessInternal (and nothing good can come off it).

Reply:
Ack, this makes sense.